packages/website/src/components/playground/OptionsSelector.tsx

```diff
@@ -0,0 +1,147 @@
+
+  const [copyLink, copyLinkToClipboard] = useClipboard(() =>
+    document.location.toString(),
+  );
+  const [copyMarkdown, copyMarkdownToClipboard] = useClipboard(() =>
+    createMarkdown(config),
+  );
+
+  const openIssue = useCallback((): void => {
+    const params = createMarkdownParams(config);
+
+    window
+      .open(
+        `https://github.com/typescript-eslint/typescript-eslint/issues/new?${params}`,
+        '_blank',
+      )
+      ?.focus();
+  }, [config]);
+
+  return (
+    <>
+      <Expander label="Info">
+        <InputLabel name="TypeScript">
+          <Dropdown
+            name="ts-version"
+            options={tsVersions}
+            value={config.ts}
+            onChange={(ts): void => setConfig({ ts })}
+          />
+        </InputLabel>
+        <InputLabel name="Eslint">{process.env.ESLINT_VERSION}</InputLabel>
+        <InputLabel name="TSEslint">{process.env.TS_ESLINT_VERSION}</InputLabel>
+      </Expander>
+      <Expander label="Options">
+        <InputLabel name="Source type">
+          <Dropdown
+            name="sourceType"
+            value={config.sourceType ?? 'module'}
+            onChange={(sourceType): void => setConfig({ sourceType })}
+            options={['script', 'module']}
+          />
+        </InputLabel>
+        <InputLabel name="File type">
+          <Dropdown
+            name="fileType"
+            value={config.fileType ?? 'ts'}
+            onChange={(fileType): void => setConfig({ fileType })}
+            options={fileTypes}
+          />
+        </InputLabel>
+        <InputLabel name="Auto scroll">
+          <Checkbox
+            name="enableScrolling"
+            value=""
+            checked={enableScrolling}
+            onChange={setEnableScrolling}
+          />
+        </InputLabel>
+      </Expander>
+      <Expander label="Actions">
+        <ActionLabel name="Copy link" onClick={copyLinkToClipboard}>
+          <Tooltip open={copyLink} text="Copied">
+            <CopyIcon width="13.5" height="13.5" />
+          </Tooltip>
+        </ActionLabel>
+        <ActionLabel name="Copy Markdown" onClick={copyMarkdownToClipboard}>
+          <Tooltip open={copyMarkdown} text="Copied">
+            <CopyIcon width="13.5" height="13.5" />
+          </Tooltip>
+        </ActionLabel>
+        <ActionLabel name="Report as Issue" onClick={openIssue}>
+          <IconExternalLink width="13.5" height="13.5" />
+        </ActionLabel>
+      </Expander>
+      {windowSize !== 'mobile' && (
+        <Expander label="Debug">
+          <InputLabel name="vfs instance">
+            <code>window.system</code>
+          </InputLabel>
+          <InputLabel name="monaco instance">
+            <code>window.monaco</code>
+          </InputLabel>
+          <InputLabel name="typescript">
+            <code>window.ts</code>
+          </InputLabel>
+          <InputLabel name="esquery">
+            <code>window.esquery</code>
+          </InputLabel>
```

Comment:
Nit: it's been confusing my brain that these have `cursor: pointer` but don't do anything on click. Can that be removed? 😄

Reply:
this is only useful during dev, and shoudl be removed before merge